cartography/intel/aws/iam.py

```diff
@@ -718,6 +718,33 @@ def _load_policy_tx(
         aws_update_tag=aws_update_tag,
     )
 
+    ingest_sso_policy = """
```

Comment:
we are inserting the same policy again?

Reply:
in sso scenario we mapping the principal with id field and in non sso scenario mapping with arn, that’s why we need to separate queries